mpa/cls/trainer.py

```diff
@@ -158,7 +160,8 @@ def train_worker(gpu, dataset, cfg, distributed, validate, timestamp, meta):
                         num_gpus=len(cfg.gpu_ids),
                         dist=distributed,
                         round_up=True,
-                        seed=cfg.seed
+                        seed=cfg.seed,
+                        drop_last=train_data_cfg.drop_last if train_data_cfg.get('drop_last', False) else False
```

Comment:
Is there a reason to add this?

Reply:
Yes, default batch size for classification task is 32 and the minimum sample number for SC classification is only 18.
In this case, if we set drop_last=True, the image never income to pass forward. (only dry-run). I handled this case in task.py (https://github.com/openvinotoolkit/training_extensions/blob/jeom/enable-mpa-hierarchical/external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py#L280)

But with hierarchical classification, this task requires drop_last=True, for stable hierarchical information indexing.
So, I add this capacity to avoid the situation I mentioned above.